src/turtle/runner/Runner.d

```diff
@@ -614,6 +614,10 @@ class TurtleRunner ( TaskT ) : CliApp
     {
         SchedulerConfiguration config;
         initScheduler(config);
+        // clear exception handler to workaround regression caused by ocean
+        // v3.6.0 and later which causes test suite to abort during termination
+        // of the tested app because of unhandled exception:
+        theScheduler.exception_handler = null;
```

Comment:
So with makd v2.x.x, this isn't a problem?

Reply:
Yes, because it doesn't compile integration tests with `-version=UnitTest`.

Reply:
What I'm wondering is what we do with this workaround in the future. Presumably it should be removed at some point. When? How do we remember to do this?

Reply:
I don't think we can ever remove it. Not until swarm design stops needing that hack (expecting uncaught exceptions in the event loop) in ocean to work :)

Reply:
I don't see how swarm is the problem. As you said, building with makd v2.x.x removes this issue. So it seems to me that when makd v1.x.x is no longer supported, we could remove this workaround.